### ISSUE TITLE
Error when trying to run any ngxs schematics

### ISSUE THREAD

**Issue by chungtc** [nrwl/nx#15144]
### Current Behavior

I'm getting this error when I try to run any @ngxs/schematics (i.e nx g se Selectorz)

```
$> nx g se Selectorz --verbose

>  NX  Generating @ngrx/schematics:selector

SynchronousDelegateExpectedException [Error]: Expected a synchronous delegate but got an asynchronous one.
    at SyncDelegateHost._doSyncCall (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/core/src/virtual-fs/host/sync.js:39:19)
    at SyncDelegateHost.read (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/core/src/virtual-fs/host/sync.js:57:21)
    at LazyFileEntry._load (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/schematics/src/tree/host-tree.js:252:80)
    at get content [as content] (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/schematics/src/tree/entry.js:34:55)
    at HostTree.read (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/schematics/src/tree/host-tree.js:211:30)
    at getWorkspace (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@ngrx/schematics/schematics-core/utility/config.js:13:29)
    at getProject (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@ngrx/schematics/schematics-core/utility/project.js:6:47)
    at getProjectPath (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@ngrx/schematics/schematics-core/utility/project.js:19:19)
    at /tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@ngrx/schematics/src/selector/index.js:18:61
    at callRuleAsync (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/schematics/src/rules/call.js:78:24)

 >  NX   Expected a synchronous delegate but got an asynchronous one.
```

### Expected Behavior

1. Expect to be able to use ngxs schematics

### Steps to Reproduce

1. yarn create nx-workspace --preset=angular-monorepo
2. yarn add @ngrx/{store,entity,effects}@latest
3. yarn
4. nx serve
5. nx g container Compo  // <-- or any other ngxs schematics

### Nx Report

```shell
>  NX   Report complete - copy this into the issue template

   Node : 18.14.1
   OS   : linux x64
   yarn : 1.22.19
   
   nx                      : 15.7.2
   @nrwl/jest              : 15.7.2
   @nrwl/linter            : 15.7.2
   @nrwl/workspace         : 15.7.2
   @nrwl/angular           : 15.7.2
   @nrwl/cli               : 15.7.2
   @nrwl/cypress           : 15.7.2
   @nrwl/devkit            : 15.7.2
   @nrwl/eslint-plugin-nx  : 15.7.2
   @nrwl/js                : 15.7.2
   @nrwl/tao               : 15.7.2
   @nrwl/webpack           : 15.7.2
   typescript              : 4.8.4
   ---------------------------------------
   Community plugins:
   @ngrx/component-store : 15.3.0
   @ngrx/effects         : 15.3.0
   @ngrx/entity          : 15.3.0
   @ngrx/router-store    : 15.3.0
   @ngrx/store           : 15.3.0
   @ngrx/store-devtools  : 15.3.0
   @ngrx/schematics      : 15.3.0
```

### Failure Logs

```shell
$> nx g se Selectorz --verbose

>  NX  Generating @ngrx/schematics:selector

SynchronousDelegateExpectedException [Error]: Expected a synchronous delegate but got an asynchronous one.
    at SyncDelegateHost._doSyncCall (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/core/src/virtual-fs/host/sync.js:39:19)
    at SyncDelegateHost.read (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/core/src/virtual-fs/host/sync.js:57:21)
    at LazyFileEntry._load (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/schematics/src/tree/host-tree.js:252:80)
    at get content [as content] (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/schematics/src/tree/entry.js:34:55)
    at HostTree.read (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/schematics/src/tree/host-tree.js:211:30)
    at getWorkspace (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@ngrx/schematics/schematics-core/utility/config.js:13:29)
    at getProject (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@ngrx/schematics/schematics-core/utility/project.js:6:47)
    at getProjectPath (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@ngrx/schematics/schematics-core/utility/project.js:19:19)
    at /tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@ngrx/schematics/src/selector/index.js:18:61
    at callRuleAsync (/tdev/learn/ngrx/ngrx-test-standalone-nx/node_modules/@angular-devkit/schematics/src/rules/call.js:78:24)

 >  NX   Expected a synchronous delegate but got an asynchronous one.
```

### Additional Information

i've tried creating different types of nx workspace (monorepo / standalone) all giving me same error

**Comment by AgentEnder** [comment]
I think this is functionally a duplicate of #15120, which I believe to be fixed by #15137. @leosvelperez will confirm.

**Comment by leosvelperez** [comment]
That's correct. This was fixed in #15137 and the fix will be released soon. Keep an eye on the releases to get it.
